nrf_security/src/legacy/CMakeLists.txt

```diff
@@ -127,7 +154,7 @@ if(TARGET mbedcrypto_cc3xx_imported)
   )
 
   # Add cc3xx platform/mbedcrypto includes to mbedcrypto library
-  target_include_directories(${mbedcrypto_target}
+  target_include_directories(mbedcrypto_common
```

Comment:
When changing the name of the target, remember to update the comment accordingly.